packages/react-button/src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
+import { Button, ButtonProps } from '../../Button';
+import { Meta } from '@storybook/react';
+
+// TODO: this is here while waiting for react-icons to merge
+import { CalendarIcon } from '../../tmp-icons.stories';
+
+export const Default = (props: ButtonProps) => <Button {...props}>Button</Button>;
+
+export const Emphasis = () => (
+  <>
+    <Button primary>Primary button</Button>
+    <Button>Default button</Button>
+    <Button outline>Outline button</Button>
+    <Button subtle>Subtle button</Button>
+    <Button transparent>Transparent button</Button>
+  </>
+);
+Emphasis.parameters = {
+  docs: {
+    description: {
+      story:
+        '- `primary` button is used for the most important action on the page or in a view\n' +
```

Comment:
instead of backtick newlines and concatenation, would it be possible to use template strings ?

Reply:
I've tried using template strings, but couldn’t find a way to make it better.

If we use template strings and format the code nicely, the indentation on doc page will be wrong:

<img width="663" alt="Screenshot 2021-08-10 at 09 40 43" src="https://user-images.githubusercontent.com/18402223/128828008-98c7cdde-4e5a-4482-912f-b86b3b73cff7.png">

The only way I could get template literals to produce correct list indentation is without indentation in code:
```tsx
Emphasis.parameters = {
  docs: {
    description: {
      story: `
- \`primary\` button is used for the most important action on the page or in a view
- \`default\` button is used for subordinate actions
- \`outline\` has no background styling and is emphasized through the styling of its content and borders
- \`transparent\` has no background or border styling and is just emphasized through its content styling
- \`subtle\` button blends into its background and becomes less emphasized`,
    },
  },
};
```

I feel like backtick newlines and concatenation is a nicer solution.

Reply:
That sounds good, thx for trying it out 👍